ui/component/header/view.jsx

```diff
@@ -324,26 +325,26 @@ const Header = (props: Props) => {
                       )}
                     </MenuButton>
                     <MenuList className="menu__list--header">
-                      <MenuItem className="menu__link" onSelect={() => history.push(`/$/${PAGES.UPLOADS}`)}>
+                      <MenuLink className="menu__link" as={Link} to={`/$/${PAGES.UPLOADS}`}>
```

Comment:
Trying to middleclick the `<MenuItem>` with other props in set won't work! I tried it already to introduce very little changes, but no luck.

You press mouse wheel on `<MenuItem>` with `as={Link} to={...}` - it just navigates in the current tab. Undesirable. So, `<MenuLink>` is way more appropriate here. [Reach docs tell so as well...](https://reach.tech/menu-button/#menulink)